common.blocks/menu/menu.js

```diff
@@ -135,11 +148,19 @@ provide(BEMDOM.decl({ block : this.name, baseBlock : Control }, /** @lends menu.
      */
     _onItemHover : function(item) {
         if(item.hasMod('hovered')) {
+            var id = this._activedescendant;
+
             this._hoveredItem && this._hoveredItem.delMod('hovered');
+            item.domElem.attr('id', id);
+            this.domElem.attr('aria-activedescendant', id);
             this._scrollToItem(this._hoveredItem = item);
         } else if(this._hoveredItem === item) {
+            item.domElem.removeAttr('id');
+            this.domElem.removeAttr('aria-activedescendant');
             this._hoveredItem = null;
         }
+
+        this.emit('item-hover');
```

Comment:
Let's make this change in `select`'s PR. So we will see, how it used and try to find the way to avoid it.